packages/renderer-protocol/inverseRpc/services/friendRequestService.ts

```diff
@@ -72,7 +73,32 @@ export function registerFriendRequestKernelService(port: RpcServerPort<RendererP
     },
 
     async cancelFriendRequest(req, _) {
-      return {}
+      try {
+        // Handle cancel friend request
+        const cancelFriend = await cancelFriendRequest(req)
+
+        let cancelFriendRequestReply: CancelFriendRequestReply = {}
+
+        // Check the response type
+        if (cancelFriend.reply?.friendRequest) {
+          cancelFriendRequestReply = {
+            message: {
+              $case: 'reply',
+              reply: cancelFriend.reply
+            }
+          }
```

Comment:
Wondering whether we can do the same here as with the error function for all replies alike.

Reply:
Totally! Take a look at what I have managed to carry out (https://github.com/decentraland/kernel/pull/746/commits/bc24a1b692c22c112b70d1763634234baf234f98)